specification/cognitiveservices/OpenAI.Inference/client.tsp

```diff
@@ -19,3 +19,27 @@ using Azure.ClientGenerator.Core;
 @@usage(Azure.OpenAI.ImageGenerationOptions, Usage.input | Usage.output);
 @@usage(Azure.OpenAI.ImageLocation, Usage.input | Usage.output);
 @@usage(Azure.OpenAI.ImageGenerations, Usage.input | Usage.output);
+
+// Java will need to have those models expose in public surface.
+@@access(Azure.OpenAI.AudioTaskLabel, Access.public, "java");
+@@access(Azure.OpenAI.AudioTranscription, Access.public, "java");
+@@access(Azure.OpenAI.AudioTranscriptionSegment, Access.public, "java");
+@@access(Azure.OpenAI.AudioTranslation, Access.public, "java");
+@@access(Azure.OpenAI.AudioTranslationSegment, Access.public, "java");
+@@access(Azure.OpenAI.AzureChatExtensionConfiguration, Access.public, "java");
+@@access(Azure.OpenAI.AzureChatExtensionType, Access.public, "java");
+@@access(Azure.OpenAI.AzureOpenAIOperationState, Access.public, "java");
+@@access(Azure.OpenAI.ChatChoice, Access.public, "java");
+@@access(Azure.OpenAI.ChatCompletions, Access.public, "java");
+
+// Error: Model 'ChatCompletionsOptions' cannot be set as access=public, as it refers Union 'FunctionCallPreset | FunctionName'
+// @@access(Azure.OpenAI.ChatCompletionsOptions, Access.public, "java");
```

Comment:
@weidongxu-microsoft 

We have customized this `ChatCompletionsOptions` class, which hides some generated methods, and introduced some new implementations to it. 

 @Generated
    public FunctionCallModelBase getFunctionCall() {
        return this.functionCall;
    }

should replaced by 
  public FunctionCallConfig getFunctionCall() {
        return this.functionCallConfig;
    }